go.mod

```diff
@@ -4,7 +4,7 @@ go 1.15
 
 require (
 	cloud.google.com/go/storage v1.13.0
-	contrib.go.opencensus.io/exporter/stackdriver v0.12.1
+	contrib.go.opencensus.io/exporter/stackdriver v0.13.5
```

Comment:
If this PR can work with old versions of dependencies, it's better to keep it and update go.[mod|sum] in a separate PR

Reply:
Directly copied and replace the go.mod and go.sum file from minikube master branch will resolve this issue.